tool/generator/translator.dart

```diff
@@ -344,7 +345,17 @@ class _PartialInterfacelike {
           if (attribute.special == 'static') {
             staticMembers.add(member);
           } else {
-            members.add(member);
+            if (name == 'SVGElement' && attribute.name == 'className') {
```

Comment:
Part of me wants to scream: "generalize this!" but maybe we wait for rule of three.

If we have a couple more special cases, we should probably come up with some `provider` or something to encapsulate these things...

Reply:
Interestingly, in `dart:svg` we hid this getter (we renamed it to `_svgClassName` and didn't expose it elsewhere). We did so with side-car info to the generator, which I guess is not that different than this.

Reply:
That should show you how long it's been deprecated but not removed. :)

I considered not exposing it, but that made less sense because that is indeed what the DOM exposes. If we decide to avoid generating deprecating members in this package as part of our versioning decisions, this special-casing can be removed.